onnx_tf/frontend.py

```diff
@@ -310,6 +332,9 @@ def tensorflow_graph_to_onnx_model(cls,
       List or tuple items should be (str domain, int version number).
     :param producer_name: The name of the producer.
     :param graph_name: The name of the output ONNX Graph.
+    :param ignore_unimplemented: Convert to ONNX model with ignoring unimplemented operators.
```

Comment:
Can we say, 

"Convert to ONNX model and ignore all the operators that are not currently supported by onnx-tensorflow".

Just to be a bit clearer.